tools-local/dir.props

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\dir.props" />
+
+  <PropertyGroup>
+    <BuildCustomTasksForDesktop Condition="'$(DotNetBuildFromSource)' != 'true' OR '$(MSBuildRuntimeType)' != 'Core'">true</BuildCustomTasksForDesktop>
```

Comment:
Why do we even care about the DotNetBuildFromSource flag? I would just base it on MsBuildRuntimeType.

Reply:
Good point.  changing.